sixtyfps_runtime/corelib/graphics/image.rs

```diff
@@ -54,39 +270,80 @@ impl Image {
     pub fn load_from_path(path: &std::path::Path) -> Result<Self, LoadImageError> {
         Ok(Image(ImageInner::AbsoluteFilePath(path.to_str().ok_or(LoadImageError(()))?.into())))
     }
-    /*
-    /// Create an image from argb pixels.
+
+    /// Creates a new Image from the specified shared pixel buffer, where each pixel has three color
+    /// channels (red, green and blue) encoded as u8.
+    pub fn new_rgb8(buffer: SharedPixelBuffer<RGB8Pixel>) -> Self {
```

Comment:
should this function and other maybe be called  `from_rgb8`   ?

Reply:
Okay, I'll rename them. It makes sense, as without the `Premultiplied` distinction we'd use the `From` trait I think.